### DISCUSSION TITLE
Noi v0.2.1

### DISCUSSION THREAD

**Discussion by lencx** [lencx/Noi#10] [Announcements]
Fix: Windows cannot work.

<hr /><em>This discussion was created from the release <a href='https://github.com/lencx/Noi/releases/tag/v0.2.1'>Noi v0.2.1</a>.</em>

**Comment by babyonebyone** [comment]
windows cant  work

**Comment by zhougy0717** [comment]
mac arm cant work

**Reply by lencx** [reply]
https://github.com/lencx/Noi#macos
